tensorflow/python/keras/utils/vis_utils.py

```diff
@@ -124,6 +163,18 @@ def model_to_dot(model,
     dot.set('dpi', dpi)
     dot.set_node_defaults(shape='record')
 
+  if layer_range is not None:
+    if len(layer_range) != 2:
+      raise ValueError("layer_range must be of shape (2,)")
+    if not isinstance(layer_range[0], str) or\
```

Comment:
Please don't use \ here, and use () to wrap the long line instead.

Reply:
done